src/content/packages-and-plugins/swift-package-manager/for-app-developers.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Swift Package Manager for app developers
+description: How to use Swift Package Manager for native iOS or macOS dependencies
+---
+
+Flutter is migrating to Swift Package Manager to manage iOS and macOS native
+dependencies.
+This is an experimental feature that may change in the future.
+It is currently only available on the [`master` channel][].
+Flutter will continue to support CocoaPods until further notice.
+
+:::tip
+If you find a bug in Flutter's Swift Package Manager feature,
+please [open an issue][].
+:::
+
+[`master` channel]: /release/upgrade#switching-flutter-channels
+[open an issue]: {{site.github}}/flutter/flutter/issues/new?template=2_bug.yml
+
+{% include docs/swift-package-manager/how-to-enable-disable.md %}
+
+## Add Flutter to an existing app (add-to-app)
+
+Flutter's Swift Package Manager feature does not support add-to-app scenarios
+yet.
+
+For status updates, see [flutter#146957].
+
+[flutter#146957]: https://github.com/flutter/flutter/issues/146957
+
+## How to add Swift Package Manager integration to a project manually
```

Comment:
Our hope is that the automatic migration will migrate most projects. Hopefully most users won't need to follow these steps.

However, users can modify their project in a way that breaks out automatic migration. Ideally users would let us know about these modifications so that we can improve our automatic migration.

Question for reviewers: Is it clear that most people can skip these steps?

Reply:
**suggestion:** Remove _manually_. If the reader is doing something, then it's manual by definition.

```suggestion
## How to add Swift Package Manager integration to a project
```

Reply:
I removed manually from the section header but clarified the section intro to clarify these steps are only necessary if the automatic migration fails.